crmprtd/moti.py

```diff
@@ -26,72 +26,92 @@
     ('extension', None, 'unitless')
 ]
 
-def makeurl(report='7110', request='historic', station=None, from_=None, to=None):
+
+def makeurl(report='7110', request='historic',
+            station=None, from_=None, to=None):
     '''
     Construct a URL for fetching the data file
     '''
     fmt = '%Y-%m-%d/%H'
     from_ = from_.strftime(fmt)
     to = to.strftime(fmt)
-    url  = 'https://prdoas2.apps.th.gov.bc.ca/saw-data/sawr{report}?request={request}&station={station}&from={from_}&to={to}'.format(**locals())
+    url = (('https://prdoas2.apps.th.gov.bc.ca/saw-data/sawr{report}?'
```

Comment:
Looks like you have changed the return type from a string to a tuple here. That's not what we want, is it?

Reply:
Or maybe that's a new multi-line string syntax that I don't know about?

Reply:
I think I accidentally put an extra set of parentheses.  The multi line string format I've been following I found [here](https://stackoverflow.com/questions/10660435/pythonic-way-to-create-a-long-multi-line-string).